src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,28 @@
 package core.basesyntax;
 
+import core.basesyntax.figures.Figure;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
```

Comment:
If you decide to use this class, you can just rename it. HelloWorld isn’t an appropriate name.

Reply:
done